packages/apputils-extension/src/index.ts

```diff
@@ -564,8 +568,13 @@ const sessionDialogs: JupyterFrontEndPlugin<ISessionContextDialogs> = {
 const utilityCommands: JupyterFrontEndPlugin<void> = {
   id: '@jupyterlab/apputils-extension:utilityCommands',
   requires: [ITranslator],
+  optional: [IMainMenu, ICommandPalette],
   autoStart: true,
-  activate: (app: JupyterFrontEnd, translator: ITranslator) => {
+  activate: (
+    app: JupyterFrontEnd,
+    translator: ITranslator,
+    palette: ICommandPalette
```

Comment:
As it is optional, it may be `null`.

```suggestion
    palette: ICommandPalette | null
```